src/main/java/com/revature/repositories/TripRepository.java

```diff
@@ -0,0 +1,37 @@
+package com.revature.repositories;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import com.revature.models.Trip;
+import org.springframework.stereotype.Repository;
+
+/**
+ * TripRepository extends JpaRepository
+ * This interface handles a variety of queries for trips
+ *
+ * @author joseavalos
+ */
+
+@Repository
+public interface TripRepository extends JpaRepository<Trip, Integer> {
+	/**
+	 * Custom query that uses the @Query annotation to select an array of trips by the driver's user id.
+	 *
+	 * @param id
+	 * @return Check {@link com.revature.services.impl.TripServiceImpl}
+	 */
+	@Query("select t from Trip t where t.driver.userId = ?1")
```

Comment:
Have either of these queries been tested? I am also curious why the second query returns a list of the trip ids, instead of just a list of the Trip objects.

Reply:
Updated to return Trip objects. Makes things a lot simpler now! Thank you very much for your input there, too!